src/Components/HexImage/styles.scss

```diff
@@ -0,0 +1,65 @@
+.hexagon {
+    position: relative;
+    width: 300px;
+    height: 173.21px;
+    margin: 86.6px 0;
+    background-position: center;
+    background-size: auto 346.4102px;
+}
+
+.hexTop,
```

Comment:
You should use hyphen instead of camel case for css class names.

Reply:
@kumardennis make this as a separate PR to master, this branch was merged.